### ISSUE TITLE
Simulator not showing up

### ISSUE THREAD

**Issue by DavidWhaleMEF** [Microsoft/pxt#2026]
microbit.org-ticket: 1673

The simulator is not showing up.

I have tried opening it using the jumpto.cc/pxt-new (from the Code Club website) using IE version 11 and Google Chrome version 57 with the same result.
 
We use Windows 7.

![Uploading Screen Shot 2017-05-03 at 10.42.57.png…]()

**Comment by DavidWhaleMEF** [comment]
[Uploading Screen Shot 2017-05-03 at 10.42.57.zip…]()

**Comment by DavidWhaleMEF** [comment]
[Uploading Screen Shot 2017-05-03 at 10.42.57.zip…]()

Sorry, github image upload has broken, I'll describe the screen.

A normal PXT screen, with ON START and FOREVER on right with no code in them.

On the left, the white patterned background with the STOP button etc (so simulator is running)

But no simulator. There is a gap where the simulator would be.

**Comment by tballmsft** [comment]
Doesn't repro for me. on Windows 10.

**Comment by abchatra** [comment]
@DavidWhaleMEF  Please provide a repro. This is not actionable. Likely there is a simulator crashed for some reason.

**Comment by DavidWhaleMEF** [comment]
I have a screen dump from the user, just trying to upload it now. 

I can't repro this myself either. But it doesn't work for the user.

Are there additional steps we could suggest to the user so they could send us information in addition to a screen dump - a console log or something from the developer tools, would that help?

**Comment by abchatra** [comment]
Can you explain what is jumpto.cc/pxt-new

**Comment by DavidWhaleMEF** [comment]
![crash](https://cloud.githubusercontent.com/assets/25079586/25693416/0d57879a-30a1-11e7-90d3-5e9f00aeb97d.png)

**Comment by DavidWhaleMEF** [comment]
jumpto.cc/pxt-new is just a url redirector from CodeClub I think. I am suspecting that they put it in all their printed resources, so that if(when) the main domain/URL changes, they don't have to reprint all their resources.

**Comment by DavidWhaleMEF** [comment]
Yes, it is just a URL redirector. Code club have online resources and all of them have a 'download as PDF' option. The thousands of code club volunteers around the country then click on the PDF link and print out the resources, to take into their code club lessons and use directly with children. So I'm pretty sure I'm right in saying that a URL/domain change would be rather inconvenient for them, so they have done the typical computer science thing and 'adding an extra level of indirection solves all problems' :)

Here are details of a sample resource they have written for their code club volunteers around the country to use: 

https://codeclubprojects.org/en-GB/microbit/interactive-badge/

![screen shot 2017-05-04 at 08 12 45](https://cloud.githubusercontent.com/assets/25079586/25693497/83a6cbae-30a1-11e7-97eb-65b5db7c2724.png)

**Comment by DavidWhaleMEF** [comment]
P.S. we have general advice over here on domains that need to be unblocked in order for the PXT editor to work. If you could additionally review those for us and let me know if any domains are missing (fonts are usually one that kills things if they are stored externally), as that might additionally be a cause.

https://support.microbit.org/solution/articles/19000030385-firewall-whitelist-requirements-for-micro-bit-editors

**Comment by DavidWhaleMEF** [comment]
If it would be possible for us both to write some diagnostics instructions for users explaining how to get a decent debug log we can send you, that would help here too. I'll log a new ticket suggesting this, perhaps we can work together on it.

**Comment by abchatra** [comment]
Here is the right set of domains: https://pxt.microbit.org/faq
makecode.com is missing in the list of required domains to unblock

**Comment by pelikhan** [comment]
Isn’t it the second listed domain?

**Comment by abchatra** [comment]
MakeCode.com is missing here:
https://support.microbit.org/solution/articles/19000030385-firewall-whitelist-requirements-for-micro-bit-editors

@DavidWhaleMEF Can you just point to us in your FAQ?

**Comment by DavidWhaleMEF** [comment]
makecode.com added to kbase

We want to list all our editor requirements in one place, rather than having users hop around. So because we also list python, this is the definitive list of all whitelisting. This is so that teachers can direct their IT department at a single page of instructions and get them correctly installed without the chance that one is missed.

If you update your domains, just let us know, and we'll update our kbase to follow.

**Comment by tballmsft** [comment]
@abchatra - anything left to be done here?

**Comment by abchatra** [comment]
No action on us. Closing this issue.
